### PR TITLE
Update tests to reflect new data

### PR DESCRIPTION
**Pull request by rmarianski**
@zerebubuth could you review please?

### CODE REVIEW

File: test/592-add-adjust-bicycle-pois.py [hunk 1]
```diff
@@ -22,8 +22,8 @@
 assert_has_feature(
     16, 17238, 25950, 'pois',
     { 'kind': 'bicycle_rental_station', 'min_zoom': 17,
-      'capacity': 19, 'network': 'Bike Chattanooga',
-      'operator': 'Alta Bike Share', 'ref': '5' })
+      'capacity': 15, 'network': 'Bike Chattanooga',
+      'operator': 'Alta Bike Share', 'ref': '24' })
```
Comment (zerebubuth) [inline test/592-add-adjust-bicycle-pois.py]: I'm still seeing `capacity=19` and `ref=5` on [the node](http://www.openstreetmap.org/node/1960994139). Is it possible that it didn't get an `mz_poi_min_zoom` set for some reason?

Comment (rmarianski) [inline test/592-add-adjust-bicycle-pois.py]: Nice catch. I got confused with [this node](http://www.openstreetmap.org/node/1960994177), which is showing up in the dev tile and looked similar.

Comment (rmarianski) [inline test/592-add-adjust-bicycle-pois.py]: Turns out that this was being caused by the `remove_duplicate_features` transform removing it. This test failure should get resolved with: https://github.com/mapzen/vector-datasource/pull/772

**Comment by rmarianski** [comment]
@zerebubuth I removed the commit to update the pois tests.

**Comment by zerebubuth** [comment]
:+1: looks good!

**Pull request merged**